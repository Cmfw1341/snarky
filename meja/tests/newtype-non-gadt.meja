let f = fun (type t, x : t) => {
  switch (x) {
    | 1 => 15
    | _ => 20
  };
};
