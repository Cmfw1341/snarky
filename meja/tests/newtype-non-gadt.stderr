File "tests/newtype-non-gadt.meja", line 3, characters 6-7:
Error: Cannot unify t and int
